docs/protocol.md

```diff
@@ -1,18 +1,20 @@
-<a href="../api/protocols/base#protocol-objects">`Protocols`</a> define agent to agent interactions, which include:
+<a href="../api/protocols/base#protocol-objects">`Protocols`</a> define the structure of agent-to-agent and component-to-component interactions. Interactions in the AEA world are in the form of communication. This is influenced by established practices in the field of multi-agent systems and the prominent speech-act theory which suggests that a communicative expression is not only about transferring information from the speaker to the hearer, but that there may be meanings and commitments beyond the statement's appearance. Therefore, speech may more suited be considered as action. For example, "I hereby appoint you as chairman" is not just a sequence of words, but an action done by the speaker with wide-ranging consequences for the hearer and any other audience to that sentence.
```

Comment:
We have a separate page which explains what interactions are. Note, we must be careful not to confuse the user as `interactions = protocols + skills (+connections for transport)`

Reply:
This text here is way too detailed for the first para imo.